importer.go

```diff
@@ -0,0 +1,78 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"log"
+
+	tfe "github.com/hashicorp/go-tfe"
+	"github.com/hashicorp/terraform-exec/tfexec"
+)
+
+type Importer struct {
+	e *tfexec.Terraform
+	c *tfe.Client
+}
+
+func NewImporter(tf *tfexec.Terraform, token string, host string) (*Importer, error) {
+	tfc, err := tfe.NewClient(&tfe.Config{
+		Token:   token,
+		Address: fmt.Sprintf("https://%s", host),
+	})
+
+	if err != nil {
+		return nil, err
+	}
+
+	return &Importer{
+		tf,
+		tfc,
+	}, nil
+}
+
+func (tf *Importer) ImportWorkspace(ctx context.Context, name string, organization string, opts ...tfexec.ImportOption) error {
+	address := fmt.Sprintf("tfe_workspace.workspace[\"%s\"]", name)
+
+	state, err := tf.e.Show(context.Background())
```

Comment:
```suggestion
	state, err := tf.e.Show(ctx)
```

Couple other instances where ctx should be used as well instead of the background context.